Dockerfile

```diff
@@ -3,7 +3,9 @@ WORKDIR $GOPATH/avalanche
 COPY . .
 RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -o=/bin/avalanche ./cmd
 
-FROM scratch
+FROM alpine:latest 
```

Comment:
What are you gaining from changing the dockerfile? It seems a bit orthogonal from the rest of the change.

Reply:
I need to add the ca-certificate module to establish tls communication with my remote targets.